takwimu/templates/takwimu/_includes/navbar/_dropdown.html

```diff
@@ -6,7 +6,7 @@
 
 <div class="dropdown-menu-lg text-light py-5 container-fluid {% block dropdown_class %}{% endblock %}"
     id="{% block dropdown_id %}{% endblock %}"
-    style="{% block dropdown_style %}{% endblock %}">
+    style="{% block dropdown_style %}{% endblock %} box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.24);">
```

Comment:
The `shadow` class doesn't do this for us?

Reply:
@DavidLemayian the `shadow` class give us `box-shadow: 0 .5rem 1rem rgba(0,0,0,.15) !important;`
The zeplin design is `box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.24)`
Should use the `shadow` class instead ?

Reply:
@mtuchi Yes, let's use shadow class instead for standard look and feel with Bootstrap right now.